### ISSUE TITLE
Hard to see the focus highlight on a red background

### ISSUE THREAD

**Issue by phet-steele** [phetsims/resistance-in-a-wire#118]
Not sure we would ever have another sim with a background color like this, but when the 'R' takes up the entire screen, the contrast between the background and focus highlight is minimal. This only affects the focus highlight when on the Reset All button:

<img width="1228" alt="screen shot 2017-11-27 at 11 36 24 am" src="https://user-images.githubusercontent.com/5863899/33282927-71773f22-d367-11e7-8b8f-bec90322324e.png">

Seen on macOS 10.13.1 Chrome. For phetsims/QA/issues/73.

<details>
<summary>Troubleshooting Information</summary>

URL: https://www.colorado.edu/physics/phet/dev/html/resistance-in-a-wire/1.4.0-rc.2/resistance-in-a-wire_en.html
Version: 1.4.0-rc.2 2017-11-27 18:13:32 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36
Language: en-US
Window: 1920x1013
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 32 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"HEAD"},"axon":{"sha":"b8dd4e51","branch":"HEAD"},"babel":{"sha":"6174c331","branch":"master"},"brand":{"sha":"cfca902d","branch":"HEAD"},"chipper":{"sha":"b8a3c09a","branch":"HEAD"},"dot":{"sha":"84bc5146","branch":"HEAD"},"joist":{"sha":"ab75c309","branch":"HEAD"},"kite":{"sha":"91e2ccf1","branch":"HEAD"},"phet-core":{"sha":"5b0df3e8","branch":"HEAD"},"phetcommon":{"sha":"5b0c8dd6","branch":"HEAD"},"query-string-machine":{"sha":"c74e454e","branch":"HEAD"},"resistance-in-a-wire":{"sha":"b3b0c5c7","branch":"HEAD"},"scenery":{"sha":"db7ea875","branch":"HEAD"},"scenery-phet":{"sha":"157e5285","branch":"HEAD"},"sherpa":{"sha":"0d590744","branch":"HEAD"},"sun":{"sha":"79c0dece","branch":"HEAD"},"tandem":{"sha":"513bc954","branch":"HEAD"}}
</details>

**Comment by terracoda** [comment]
You have found and interesting in-context contrast issue, @phet-steele.

The same issue might exist in Ohm's Law.

**Comment by terracoda** [comment]
It exists half-way in Ohm's Law:
<img width="477" alt="screen shot 2017-11-27 at 15 30 17" src="https://user-images.githubusercontent.com/1902116/33283912-ed4320ce-d387-11e7-942e-3322cbd35b5e.png">

**Comment by terracoda** [comment]
@jessegreenberg, is there a way to address this problem by changing the focus highlight color of the background has changed?

**Comment by jessegreenberg** [comment]
@terracoda there is no way to determine when the background behind the reset all button has changed color. What do you think about changing the focus highlight color for this sim, or changing the focus highlight color for this reset all button permanently? Maybe to a light blue or something?

Also assigning to @emily-phet so she is aware and in case she has any comments.

**Comment by zepumph** [comment]
It would be pretty cool (but expensive) if scenery had a constrastListener that you could wire up between the focusHighlight and other colors in the sim (like the background). And update the focusHighlight accordingly.

**Comment by terracoda** [comment]
@jessegreenberg, I was thinking that an inner black stroke, or even possibly an inner and outer black stroke could work for the RIAW reset button.

The same or similar solution could work for the sound button and reset button in OL.
I can try to do a quick mock-up.

**Comment by terracoda** [comment]
@zepumph, very interesting idea re comment https://github.com/phetsims/resistance-in-a-wire/issues/118#issuecomment-347322298

**Comment by terracoda** [comment]
Very rough additional stroking idea. I tried a 3 px stroke inside the pink focus.

![focus-issue-reset-riaw](https://user-images.githubusercontent.com/1902116/33321819-90200ade-d421-11e7-8fb5-4577e32ffeb2.jpg)

![focus-issue-sound-ol](https://user-images.githubusercontent.com/1902116/33321830-9575555c-d421-11e7-9eb0-ba0dffe6d2eb.jpg)

**Comment by terracoda** [comment]
Perhaps if the inner bounds between the button and the pink focus was all filled with a high contrast color like black, it would look nicer?

**Comment by jessegreenberg** [comment]
For this sim, we decided to use a black outer focus
```js
  var OUTER_FOCUS_COLOR = new Color( 'black' );
```

**Comment by jessegreenberg** [comment]
Also, this will only be on the reset all button.

**Comment by jessegreenberg** [comment]
Added, change will need to be cherry-picked into  release branch.

**Comment by jessegreenberg** [comment]
Cherry-picked into 1.4, closing.
